util/util.go

```diff
@@ -90,3 +98,29 @@ func WriteProperties(writer io.Writer, props map[string]string) error {
 
 	return nil
 }
+
+// LoadPrivateKey load private key bytes into rsa.PrivateKey
+func LoadPrivateKey(privateKeyBytes []byte) (*rsa.PrivateKey, error) {
```

Comment:
How did we lose coverage of this function?

Reply:
This was indirectly covered in the `server` package. There was no tests for it. I am aware of it and planning to add tests for it in a separate PR.